examples/light-switch-app/genio/README.md

```diff
@@ -0,0 +1,286 @@
+# CHIP MT793X Light Switch Example
```

Comment:
Should probably use Matter, not CHIP, here....

Reply:
Replaced.